### ISSUE TITLE
artifactory-ha parse error with artifactory.node.replicaCount

### ISSUE THREAD

**Issue by sunkyzz** [jfrog/charts#1808]
**Is this a request for help?**: No

---

**Is this a BUG REPORT or FEATURE REQUEST?**: BUG REPORT

**Version of Helm and Kubernetes**: 

```bash
▶ helm version
version.BuildInfo{Version:"v3.12.2", GitCommit:"1e210a2c8cc5117d1055bfaa5d40f51bbc2e345e", GitTreeState:"clean", GoVersion:"go1.20.5"}

▶ kubectl version --short
Client Version: v1.25.9
Kustomize Version: v4.5.7
Server Version: v1.25.12-eks-2d98532
```

**Which chart**: `jfrog/artifactory-ha` version: 107.63.10

**Which product license** (Enterprise/Pro/oss): Enterprise

**JFrog support reference** (if already raised with support team): 

**What happened**: if `artifactory.node.replicaCount` is larger than 0, will cause Helm chart YAML parse error

**What you expected to happen**: render correctly

**How to reproduce it** (as minimally and precisely as possible):

`values.yaml`

```yaml
artifactory:
  node:
    replicaCount: 1
```

```
helm template artifactory --namespace artifactory jfrog/artifactory-ha --version 107.63.10 -f values.yaml

Error: YAML parse error on artifactory-ha/templates/artifactory-node-statefulset.yaml: error converting YAML to JSON: yaml: line 65: mapping values are not allowed in this context

Use --debug flag to render out invalid YAML
```

**Anything else we need to know**: This error happen when chart version larger than `107.63.5`, `107.59` or `107.55` can work correctly.

**Comment by shahiinn** [comment]
Thanks for letting us know about the issue. We were able to reproduce the same on our end as well, we will look further into it.
I would suggest you use `artifactory.primary.replicaCount` instead of `artifactory.node.replicaCount` as Artifactory no longer has a primary/node concept and we might soon deprecate the `artifactory.node` configs.
You can just specify the total number of replicas required under the primary replicaCount itself.

**Comment by chukka** [comment]
fixed via #1820
